src/packages/recompose/componentFromStream.js

```diff
@@ -3,8 +3,18 @@ import { createChangeEmitter } from 'change-emitter'
 import $$observable from 'symbol-observable'
 import { config as globalConfig } from './setObservableConfig'
 
-export const componentFromStreamWithConfig = config => propsToVdom =>
-  class ComponentFromStream extends Component {
+const identity = v => v
+
+export const createObservableConfig = rawConfig => ({
```

Comment:
The tests fail if this isn't here, toESObservable is undefined for a couple of the observable configs. 

Should I instead define  toESObservable in all observable configs?